src/components/button/button.ts

```diff
@@ -32,49 +43,55 @@ export class MdButton {
     setTimeout(() => { this.isMouseDown = false; }, 100);
```

Comment:
It appears to come out as a "FocusEvent", not a KeyboardEvent, so it's always false.